admin/model/brand.php

```diff
@@ -0,0 +1,71 @@
+<?php
+include('database.php');
```

Comment:
Because we're including the database file inside of each function, we don't need it here